releaseNotes/GoFeatureManagement.md

```diff
@@ -0,0 +1,15 @@
+# Microsoft Feature Management for Go
+
+[Source code][source_code] | [Package][package] | [Samples][samples]
+
+## 1.0.0-beta.1 - July 01, 2025
+
+Initial release of Feature Management for Go, including support for the following capabilities. Note that, version `1.1.0-beta.1` or later of Go provider is required for loading feature flags from Azure App Configuration.
+
+- Introduced a separate module `featuremanagement/providers/azappconfig` to load feature flags from Azure App Configuration. External dependencies are decoupled from the core `featuremanagement` module and can be installed independently as needed.
```

Comment:
- Support for feature flag providers.
- Built-in feature flag provider for Azure App Configuration by the module `featuremanagement/providers/azappconfig`.